### ISSUE TITLE
Restaurants数据集

### ISSUE THREAD

**Issue by s1162276945** [MyTHWN/MTER#2]
你好，在Yelp官网下载的Restaurants数据集显示的是2019年的，请问这个数据集跟论文中的数据集有关系吗？yelp数据集每年都会更新吗？

**Comment by MyTHWN** [comment]
Yes, the Yelp dataset is updated every year on the official sites. The dataset described in the paper is an old one. Thank you!
